launcher/src/main/go/go.mod

```diff
@@ -0,0 +1,24 @@
+module launcher
+
+go 1.23
+
+require (
+	github.com/elastic/go-sysinfo v1.15.0
+	github.com/fatih/color v1.18.0
+	github.com/kataras/jwt v0.1.12
+	github.com/spf13/afero v1.11.0
+	github.com/theckman/go-flock v0.8.1
```

Comment:
Why doesn't the `filelock` work?

Reply:
We've tried to implement this on our own but there were subtle bugs so we've decided to use of the shelf solution.